src/GameLogic/PlugIns/ChatCommands/ArgumentsBase.cs

```diff
@@ -16,16 +19,16 @@ public override string ToString()
         {
             var properties = this.GetType().GetProperties();
             var stringBuilder = new StringBuilder();
-
+            bool isFirst = true;
             foreach (var property in properties)
             {
-                stringBuilder.Append($"{property.Name}:{property.GetValue(this)}");
-
-                // TODO this could be better
-                if (property != properties.Last())
+                if (!isFirst)
```

Comment:
This is the wrong logic, it is used to not add a space at the end of the string not at the first argument

Reply:
But isn't the result the same (arguments seperated by a space)? I just turned the logic around ;-)